handler.go

```diff
@@ -189,57 +189,23 @@ func (h *Handler) serveQuery(w http.ResponseWriter, r *http.Request, u *User) {
 
 // serveWriteSeries receives incoming series data and writes it to the database.
 func (h *Handler) serveWriteSeries(w http.ResponseWriter, r *http.Request, u *User) {
-	// TODO: Authentication.
+	database := r.URL.Query().Get(":db")
+	retentionPolicy := r.URL.Query().Get("retentionPolicy")
```

Comment:
having the db and the rp in the query string is good for now. But I'm thinking we're going to want a way to write in data via JSON UDP and that would make the formats different between the two.